### PR TITLE
FINAL MVP(READY FOR DEPLOY). 

### PR DESCRIPTION
**Pull request by NicoHwpo**
add other users view. add if no cur task or if no prev task. delete collaborate; status. Add disable buttons on task activity. Add message if no useN=. Re-organized .js script. Add favicons, not  working?. Add custom scroll bar, etc...

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -12,58 +12,314 @@
...
+        document.getElementById('createTaskBtn').disabled = false;
+        document.getElementById('endTaskBtn').disabled = true;
+    }
+    else if (stat == 2) { //NO CREATE, NO FINISH BTN = NO USER LOGGED IN
+        document.getElementById('createTaskBtn').disabled = true;
+        document.getElementById('endTaskBtn').disabled = true;
+    }
+}
+
+/*FOR THE SELECT USER DROPDOWN
```
Comment (avrittrohwer) [inline src/main/webapp/script.js:317]: Should we remove this?

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -12,58 +12,314 @@
...
+            r2c3p.className = "bi bi-x-circle-fill";
+            r2c3p.style = "color: rgb(150, 60, 60)";
+        }
+        */
+        //dentro de segunda fila, crear columna 2
+        const row2Col4 = document.createElement("div");
+        row2Col4.className = "col-3 ps-5";
+        //dentro de columna 2, poner timegoal
+        /*STATUS
+        const r2c4p = document.createElement("p");
```
Comment (avrittrohwer) [inline src/main/webapp/script.js:234]: Should we remove this?

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -12,58 +12,314 @@
...
+        //dentro de columna 3, poner titulo task
+        const r1c5h = document.createElement("i");
+        r1c5h.className = "bi bi-dash-lg";
+
+        //dentro de primera fila, crear columna 4
+        const row1Col6 = document.createElement("div");
+        row1Col6.className = "col-1";
+        //dentro de columna 3, poner titulo task
+        const r1c6h = document.createElement("h6");
+        r1c6h.innerText = format_time(task.timeCreated);
```
Comment (avrittrohwer) [inline src/main/webapp/script.js:188]: Could we add a "Started at" label?

File: src/main/webapp/team.html [hunk 4]
```diff
@@ -1,30 +1,244 @@
...
+                    </form>
+                </div>
+
+            </div>
+
+        </div>
+
+    </div>
+
+    <!-- MODAL END TASK (NOT IN USE)
```
Comment (avrittrohwer) [inline src/main/webapp/team.html:189]: Should we remove the unused commented out parts?

File: src/main/webapp/team.html [hunk 5]
```diff
@@ -1,30 +1,244 @@
...
+            </div>
+
+
+
+            <div class="col-sm-12 col-md-6 col-lg-6 colur ">
+                <div class="p-3 headers">
+                    <h3>MY TEAM ACTIVITY</h3>
+                </div>
+
+                <!-- HARD CODED TASK CONTAINER
```
Comment (avrittrohwer) [inline src/main/webapp/team.html:89]: Should we remove this commented out part?

**Review by avrittrohwer — CHANGES_REQUESTED** [review]

File: src/main/webapp/script.js [hunk 6]
```diff
@@ -12,58 +12,314 @@
...
+
+
+
```
Comment (avrittrohwer) [inline src/main/webapp/script.js:191]: Also, could we display either the `task.end` if the task is completed or the time when the time goal would be met if the task is not completed?  For the time goal calculation, I did some prototyping in the console and I think the following would work.

Example task that has not been completed yet:

```javascript
{
desc: "https://github.com/avrittrohwer/software-product-sprint-2022/pull/5/files"
end: 1657993938925
start: 1657993938925
time: "30"
title: "Review PR"
}
```

```javascript
if (task.start != task.end) {
  // Display task end time
  someElement.innerText = time_format(task.end)
} else {
  // Display when time goal will be met
  d = new Date(task.start)
  d.setMinutes(d.getMinutes() + parseInt(task.time))
  someElement.innerText = time_format(d)
}
```

File: src/main/webapp/script.js (hunk 1 above)
Comment (NicoHwpo) [reply src/main/webapp/script.js:317]: Done

File: src/main/webapp/script.js (hunk 2 above)
Comment (NicoHwpo) [reply src/main/webapp/script.js:234]: Done

File: src/main/webapp/script.js (hunk 3 above)
Comment (NicoHwpo) [reply src/main/webapp/script.js:188]: Changed to task.start. Add the "Started at" label.

File: src/main/webapp/team.html (hunk 4 above)
Comment (NicoHwpo) [reply src/main/webapp/team.html:189]: Yes... Done

File: src/main/webapp/team.html (hunk 5 above)
Comment (NicoHwpo) [reply src/main/webapp/team.html:89]: Yes... Done

File: src/main/webapp/script.js (hunk 6 above)
Comment (NicoHwpo) [reply src/main/webapp/script.js:191]: Done.... 
task finished

**Review by avrittrohwer — APPROVED** [review]

**Pull request merged**